### ISSUE TITLE
How to stop Prcessing Error: Is Excel Busy?

### ISSUE THREAD

**Issue by nksingh62** [sdllc/Basic-Excel-R-Toolkit#39]
I do not know why instruction given at  R console behave differently- especially user defined functions. Sometime they work smoothly sometime they trapped in endless loop with message **Processing Error: Is Excel Busy?**. How to break this loop? Whether function behaves in ad hock manner due to buffering problem?

**Comment by duncanwerner** [comment]
The most common reason you get that error is if you are entering text in Excel.  You must finish entering text before using any COM/API functions.

**Comment by nksingh62** [comment]
When I got this error, I was not typing anything in Excel. I was shifting the focus (through cursor) from BERT console (for running same R function again and again through arrow key and enter) and selecting some cells on different sheets of Excel.

**Comment by nksingh62** [comment]
Following code may be used for reproducing the problem with Get.Range and range.to.data.frame function

**Details of computer, software and data file**
Processor: Intel Core 2 Duo
Operating System: Windows 7 (32 bit)
MS Excel 2007
BERT v 1.33
Data file: [test.xlsx ](https://drive.google.com/open?id=0BzPHmL7AE0OuemFjUzlQcjRPa2s) at drive.google.com

code 
```
for (i in 1:5) {
  lev1<- Get.Range("A1:L4730")
  print(dim(lev1))
  rm(lev1)
  }
```
shows an error message 
**Error in BERT$.Excel(16386, list(ref)) :   SET_VECTOR_ELT() can only be applied to a 'list', not a 'NULL'** 
at execution of third cycle of loop (some time at fourth also). 

If I add range.to.data.frame function in code i.e.
```
for (i in 1:5) {
  lev1<- range.to.data.frame(Get.Range("A1:L4730"))
  print(dim(lev1))
  rm(lev1)
  }
```
Then it flashes message 
**Error in BERT$.Excel(16386, list(ref)) :   SET_VECTOR_ELT() can only be applied to a 'list', not a 'NULL'**

In this situation Excel and R _console ceases to function_. On execution of any command at R consol, it flashes ,message  **"processing error: Is excel busy?"**  Excel stop only through task manger.

**Comment by duncanwerner** [comment]
I've got it, It's a memory problem.  I'll have a fix later today.  Thanks for the detailed report.

**Comment by duncanwerner** [comment]
Fixed in 447e2b51f7c95059a2d593b79feb95b37ccaff88 and release 1.36.
